src/components/RichText.tsx

```diff
@@ -247,3 +246,10 @@ function RichTextTag({
     </React.Fragment>
   )
 }
+
+export function isOnlyEmoji(text: string) {
+  return (
+    text.length <= 15 &&
+    /^[\p{Emoji_Presentation}\p{Extended_Pictographic}]+$/u.test(text)
```

Comment:
Supports large emojis like 🧑🏻‍❤️‍💋‍🧑🏼 which is `15` and captures a few more emoji like some flags 🇿🇼

Reply:
If we wanted to do this right we'd count graphemes, but unsure it's worth the perf

Reply:
Correction: 🧑🏻‍❤️‍💋‍🧑🏼 is not caught by the regex, but I'll leave the limit here anyway since even something like 🐴 is length 2.